apps/back-end/src/app.d.ts

```diff
@@ -0,0 +1,2 @@
+// Tell TS what this is
+declare const __BUILD_INFO__: string;
```

Comment:
This is more of a question- why does this get defined as a string, when it seems to be defined as an object in the Vite build config?

Reply:
Ah, good catch - that's a mistake. Although one you'd think the TS validation would catch, in the build or in the editor...  does it do that for you?

Reply:
Resolved in a fixup commit.